### ISSUE TITLE
How ton convert Cheetah format to proc format

### ISSUE THREAD

**Issue by Calpas** [European-XFEL/karabo_data#224]
Dear experts,
I'll get patterns converted by Cheetah, and I wonder if there is a script to convert it to 'proc' format? Exactly this format for example:  /gpfs/exfel/d/proc/SPB/201801/p002120/r0098/CORR-R0098-AGIPD15-S00001.h5  
Best regards

**Comment by takluyver** [comment]
I don't think we have anything that will do that with a single command. The EuXFEL format (`raw` & `proc` folders have the same basic format) is kind of awkward, driven by the way data is acquired and written. Our focus has been on providing convenient ways to access it (karabo_data/EXtra-data) and to make the data available in other formats (e.g. CXI files with virtual datasets).

**Comment by Calpas** [comment]
Dear Takluyver,
once I get raw data process by cheetah, can I use  AGIPD_1MGeometry class on cheetah output?
Regards

**Comment by takluyver** [comment]
Most likely, yes. Cheetah produces CXI files, which are a flavour of HDF5, so you can read them with h5py. You may need to reshape the data - our geometry code expects each frame as an array with shape (16, 512, 128) - 16 modules, 512 x 128 pixels each.

**Comment by Calpas** [comment]
Dear Takluyver,
thank you for your answer. I'll try AGIPD_1MGeometry on Cheetah output and let you know if I have issue.
Regards
